src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -999,8 +999,16 @@ public boolean verifyEditLabelIsActive(int questionNumber) {
         return getEditLabel(questionNumber).getAttribute("class").contains("active");
     }
     
-    public boolean verifyVisibilityMessageIsDisplayed(int questionNumber) {
-        return getVisibilityMessage(questionNumber).isDisplayed();
+    public boolean verifyVisibilityPreviewIsDisplayed(int questionNumber) {
+        WebElement visibilityPreviewDiv = getVisibilityMessage(questionNumber);
+        List<WebElement> visibilityMessages = visibilityPreviewDiv.findElements(By.cssSelector("ul > li"));
+        return visibilityPreviewDiv.isDisplayed() && !visibilityMessages.isEmpty();
+    }
+
+    public boolean isVisibilityPreviewDisplayedForNewQuestion() {
+        WebElement visibilityPreviewDiv = browser.driver.findElement(By.cssSelector("#questionTableNew .visibilityMessage"));
+        List<WebElement> visibilityMessages = visibilityPreviewDiv.findElements(By.cssSelector("ul > li"));
+        return visibilityPreviewDiv.isDisplayed() && !visibilityMessages.isEmpty();
```

Comment:
The inconsistent method name and inability to combine the two methods will be dealt with in #5808 (shortly)